lib/versioner/systemtrash_darwin.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2016 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//+build darwin,cgo
```

Comment:
We're not. Or, well, when we do we don't get this functionality. But that's why we have the mac builder already. :)